lib/logstorage/bloomfilter.go

```diff
@@ -2,6 +2,7 @@ package logstorage
 
 import (
 	"fmt"
+	bitutil "math/bits"
```

Comment:
It isn't a good idea to use aliases for packages, since it complicates investigating and reading the code. It is better to rename local variables, which clash with the package names instead.